src/components/Text/Annotated/MetadataDetailLabelValues.tsx

```diff
@@ -0,0 +1,35 @@
+import { EntityDetail } from "../../../model/AnnoRepoAnnotation.ts";
+import {
+  translateProjectSelector,
+  useProjectStore,
+} from "../../../stores/project.ts";
+
+export function MetadataDetailLabelValues(props: { details?: EntityDetail[] }) {
+  const translateProject = useProjectStore(translateProjectSelector);
+
+  console.log("MetadataDetailLabelValues", props.details);
```

Comment:
console log

Reply:
Fixed in [8988b27](https://github.com/knaw-huc/textannoviz/pull/154/commits/8988b2734d0fa14a4d9f8206dc51dc3b236b484f)